cmd/subnetcmd/stats.go

```diff
@@ -81,29 +86,129 @@ func stats(cmd *cobra.Command, args []string) error {
 	}
 
 	table := tablewriter.NewWriter(os.Stdout)
-	rows, err := buildStats(pClient, infoClient, table, subnetID)
+	rows, err := buildCurrentValidatorStats(pClient, infoClient, table, subnetID)
 	if err != nil {
 		return err
 	}
 	for _, row := range rows {
 		table.Append(row)
 	}
 	table.Render()
+
+	table = tablewriter.NewWriter(os.Stdout)
+	rows, err = buildPendingValidatorStats(pClient, infoClient, table, subnetID)
+	if err != nil {
+		return err
+	}
+
+	if len(rows) == 0 {
+		return nil
+	}
+	for _, row := range rows {
+		table.Append(row)
+	}
+	table.Render()
 	return nil
 }
 
-func buildStats(pClient platformvm.Client, infoClient info.Client, table *tablewriter.Table, subnetID ids.ID) ([][]string, error) {
+func buildPendingValidatorStats(pClient platformvm.Client, infoClient info.Client, table *tablewriter.Table, subnetID ids.ID) ([][]string, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	// 2nd param are delegators, we don't need them here?
+	pendingValidatorsIface, _, err := pClient.GetPendingValidators(ctx, subnetID, []ids.NodeID{})
+	if err != nil {
+		return nil, fmt.Errorf("failed to query the API endpoint for the pending validators: %w", err)
+	}
+
+	pendingValidators := make([]api.PermissionlessValidator, len(pendingValidatorsIface))
```

Comment:
I don't think we can merge this until blueberry rolls out

Reply:
Yep agreed. If blueberry is close, we may want to hold on.